src/System.Private.CoreLib/shared/System/Reflection/Assembly.cs

```diff
@@ -208,5 +208,9 @@ public static Assembly LoadWithPartialName(string partialName)
         public static Assembly ReflectionOnlyLoadFrom(string assemblyFile) { throw new PlatformNotSupportedException(SR.PlatformNotSupported_ReflectionOnly); }
 
         public virtual SecurityRuleSet SecurityRuleSet => SecurityRuleSet.None;
+
+        // Exists to faciliate code sharing between CoreCLR and CoreRT.
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal bool IsRuntimeImplemented() => this is RuntimeAssembly;
```

Comment:
There is no RuntimeAssembly in CoreRT CoreLib. This helper should be in the non-shared partition.